src/Runtime/Util/mmdPlayerControl.ts

```diff
@@ -82,6 +84,8 @@ export class MmdPlayerControl {
         mmdRuntime.onAnimationDurationChangedObservable.add(this._onAnimationDurationChanged);
         mmdRuntime.onAnimationTickObservable.add(this._onAnimationTick);
         audioPlayer?.onMuteStateChangedObservable.add(this._onMuteStateChanged);
+        audioPlayer?.onPlaybackRateChangedObservable.add(this._onPlaybackRateChanged);
```

Comment:
Yes, I just wrote it this way to bypass the issue, since you didn't want it in the MmdRuntime. But thanks to your explanation, I now understand your perspective on this project